backend/src/routes/api-router.ts

```diff
@@ -1,10 +1,15 @@
 // Root|/api endpoint router
 // Any routes in /api can be accessed through here
 import express from 'express';
-import checkHealth from '../controllers/health-api.controller';
+import checkHealth from '../controllers/health-api-controller';
+import swaggerUi from 'swagger-ui-express';
+import swaggerJSDoc from 'swagger-jsdoc';
+import swaggerConfig from '../config/swaggerConfig';
 
 const router = express.Router();
 
 router.route('/health').get(checkHealth);     // Simple health endpoint to check API status
+router.use('/api-docs', swaggerUi.serve);
+router.get('/api-docs', swaggerUi.setup(swaggerJSDoc(swaggerConfig)));
```

Comment:
This can be turned into one command like so
```suggestion
app.use(
  '/api-docs',
  swaggerUi.serve,
  swaggerUi.setup(
    swaggerJSDoc(swaggerConfig),
  ),
);
```

I tested it locally, but its worth checking yourself before committing anything I ever write in a GitHub comment

Reply:
This didn't work for me locally. 
Is it because it's in a router? 

The way I did was how the docs suggested for a router:
`
or if you are using Express router

const router = require('express').Router();
const swaggerUi = require('swagger-ui-express');
const swaggerDocument = require('./swagger.json');

router.use('/api-docs', swaggerUi.serve);
router.get('/api-docs', swaggerUi.setup(swaggerDocument));
`
https://www.npmjs.com/package/swagger-ui-express

Reply:
It does work when I move it to express.ts.

Does it make more sense to have it there?

Reply:
I actually made a mistake in my initial suggestion, I wrote `app.use` because it was a copy-pasted example but failed to edit it to be router.use. That's why it's good to double check ;) 
```js
router.use(
  '/api-docs',
  swaggerUi.serve,
  swaggerUi.setup(
    swaggerJSDoc(swaggerConfig),
  ),
);
```

Reply:
Ultimately the swagger tool is middleware, and would be better to have in your `express.ts` file. 

If you wanted to make this more organized from the beginning, you can create multiple routers and use them for each /route and import them into your middleware to load in. Organization is ultimately your teams decision in how you approach it, but that is my suggestion.

You can go the current approach and refactor it later if you change your mind, once more endpoints are in place

Reply:
Thank you for the review. I agree with you and I'll put swagger on `expres.ts` since it's middleware :smile:

Reply:
I also restructured the routers into multiple files. I referenced Wayfinder to get an idea on how to do this properly.